File: Data Structures & Algorithms/reverse-bits/submission-1.cpp
class Solution {
public:
    uint32_t reverseBits(uint32_t n) {
        uint32_t tmp = 0;

        for (int i = 0; i < 32; i++) {
            tmp <<= 1;
            tmp |= (n & 1);
            n >>= 1;
        }

        return tmp;
    }
};

File: Data Structures & Algorithms/plus-one/submission-1.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        s = ""
        for l in digits:
            s+=str(l)
        num = int(s)
        num+=1

        tmp = str(num)
        final  = []
        for l in list(tmp):
            final.append(int(l))

        return final
        